src/main/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -22,6 +23,9 @@ export class Eligibility {
   @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.OVER_18] })
   eighteenOrOver?: YesNoOption
 
+  @IsIn(DefendantAgeOption.all(), { message: ValidationErrors.OVER_18_DEFENDANT_REQUIRED, groups: [ValidationGroups.OVER_18_DEFENDANT] })
```

Comment:
Please rename validation group to match property, in that case it will be `DEFENDANT_AGE`